plugins/operators/__init__.py

```
@@ -1416,6 +1416,204 @@ def execute(self, ctx):
+            index_spec={"keyPattern": key_pattern, "unique": True},
+            success_result={
+                **result,
+                "message": f"Unique index conversion completed successfully for {field=}.",
+            },
+            error_result={
+                **result,
+                "message": f"Error while making index '{field}' unique.",
+            },
+        )
+
+    def _safe_collmod(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (7/5)

(R0917)